client/my-sites/checkout/checkout-thank-you/marketplace/style.scss

```diff
@@ -2,155 +2,8 @@
 @import "@wordpress/base-styles/mixins";
 
 .marketplace-thank-you__container {
-	min-height: 100vh;
+	max-width: 750px;
+	padding: 0 24px 0;
 	display: flex;
 	flex-direction: column;
```

Comment:
I suppose we could remove these styles too, given that `.marketplace-thank-you__container` only has a single child.